HHS/Head-Start-TTADP
# [TTAHUB-1885] Update training and session report permissions

## Description of change

### Update permissions for the training report (based on current understanding)

**Event permissions**
- Event owner: can add sessions, edit event, edit and delete sessions, close or suspend event
- Event collaborator: can add sessions, edit and delete sessions. Cannot edit event.
- Regional point of contact: can edit event, edit and delete sessions. Cannot add sessions or close or suspend event.

**Event visibility**
- Not started: Only owners and those added to the event can view event and sessions
- In progress: Everyone with read or read/write can view event and sessions. Everyone else can see event but only completed sessions.
- Complete & Suspended: Utter visibility

### Chart for reference:

![Screenshot 2023-07-25 at 8 46 06 AM](https://github.com/HHS/Head-Start-TTADP/assets/3288586/e87e6d9a-f240-4912-ac68-79cd2208b140)

### Additional changes

This PR also includes a few small updates requested by the UX team.
- The topics control on the session summary now has a drawer. The language for that drawer, previously ambiguous, has been updated both here and on the AR.
- The instructions on the complete summary page have been updated
- Event creator has been added as a field to the "view event" page.

## How to test

Makes sure behavior conforms to expectations. Make sure behavior is adequately tested. Confirm smaller bugfixes.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1885
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1895
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1897
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1900

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
